platform-api/src/internal/utils/api.go

```diff
@@ -22,6 +22,8 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/getkin/kin-openapi/openapi3"
```

Comment:
kin-openapi doesn't support openapi 3.1.0 as of now (but there has been a PR added very recently to support this - https://github.com/getkin/kin-openapi/pull/1102) 
We may also consider https://github.com/pb33f/libopenapi which claims to support 3.0, 3.1 and 3.2 as well.